lib/uc3-ssm.rb

```diff
@@ -228,8 +228,7 @@ def sanitize_parameter_key(key)
     # Attempt to retrieve the value from AWS SSM
     def retrieve_ssm_value(key)
       return key if @ssm_skip_resolution
-
-      @client.get_parameter(name: key)[:parameter][:value]
+      @client.get_parameter(name: key, with_decryption: true)[:parameter][:value]
```

Comment:
Yeah, this is fine. The docs say that it ignores it if the parameter is not a SecureString